shared/reducers/chat2.tsx

```diff
@@ -787,11 +787,7 @@ const reducer = Container.makeReducer<Actions, Types.State>(initialState, {
         if (lookupID) {
           const map = oldMessageMap.get(conversationIDKey)
           const oldMsg = map?.get(Types.numberToOrdinal(lookupID))
-          if (
-            oldMsg?.type === 'placeholder' &&
-            // don't delete the placeholder if we're just about to replace it ourselves
```

Comment:
unclear why we did this. how this function works in general is way to mysterious and is a big wart on this whole flow. without this its easy for me to go into empty threads and see a placeholder that never resolves due to this check